WooCommerce/UITestsFoundation/Screens/Login/PrologueScreen.swift

```diff
@@ -3,37 +3,41 @@ import XCTest
 
 public final class PrologueScreen: ScreenObject {
 
+    private let continueButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Prologue Continue Button"]
+    }
+
+    private let selectSiteButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Prologue Self Hosted Button"]
+    }
+
+    private var continueButton: XCUIElement { continueButtonGetter(app) }
+    private var selectSiteButton: XCUIElement { selectSiteButtonGetter(app) }
+
     public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
-            expectedElementGetter: { Self.findContinueButton(in: $0) },
+            expectedElementGetter: continueButtonGetter,
```

Comment:
WDYT of including both getters?

```suggestion
            expectedElementGetters: [
                continueButtonGetter,
                selectSiteButtonGetter
            ],
```